src/content/docs/supporters/get-started.mdx

```diff
@@ -0,0 +1,223 @@
+</details>
+
+## Install the extension
+
+<LinkButton 
+  href="https://chromewebstore.google.com/detail/web-monetization/oiabcfomehhigdepbbclppomkhlknpii"
+  target="_blank"
+  variant="primary"
+  icon="external"
+  iconPlacement="start"
+>
+  Google Chrome
+</LinkButton>
+<LinkButton 
+  href="https://addons.mozilla.org/en-US/firefox/addon/web-monetization-extension/"
+  target="_blank"
+  variant="primary"
+  icon="external"
+  iconPlacement="start"
+>
+  Mozilla Firefox
+</LinkButton>
+<LinkButton 
+  href="https://microsoftedge.microsoft.com/addons/detail/web-monetization/imjgemgmeoioefpmfefmffbboogighjl"
+  target="_blank"
+  variant="primary"
+  icon="external"
+  iconPlacement="start"
+>
+  Microsoft Edge
+</LinkButton>
+
+:::note[Permissions]
+The extension asks for permission to access your browser tabs and your data for all websites. The extension requires these permissions to check each page you visit for Web Monetization.
+:::
+
+## Configure the extension
+
+After you install the extension, a page appears to guide you though the configuration steps.
+
+<Steps>
+  1. Get a Web Monetization-compatible [wallet](/wallets).<br/><img src='/img/docs/extension/onboarding-1.png' alt='An image of the onboarding screen showing the four steps to configure the extension' style='max-width:300px' class='img-outline'/>
+      :::note
+      You may experience a delay before you can fund your wallet while your provider performs any necessary regulatory checks.
+      :::
+  2. Find and copy your wallet address or payment pointer. You'll need it for Step 4. Your wallet provider will assign one to you during sign up. The exact naming depends on what your provider has chosen to use.<br/><img src='/img/docs/extension/onboarding-2.png' alt='The Interledger Wallet dashboard displaying a wallet address' style='max-width:400px' class='img-outline'/>
+      <Tabs>
+      <TabItem label='Example wallet address'>
+        ``` 
+        https://wallet.example.com/alice
+        ```
+      </TabItem>
+      <TabItem label='Example payment pointer'>
+        ```
+        $wallet.example.com/alice
+        ```
+      </TabItem>
+      </Tabs>
+  3. Pin the extension to your toolbar (optional).<br/><img src='/img/docs/extension/onboarding-3.png' alt='An example of where to find the pin to toolbar option for the extension' style='max-width:300px' class='img-outline'/>
+  4. Enter the wallet address or payment pointer you found in Step 2.<br/><img src='/img/docs/extension/onboarding-4a.png' alt='Set up screen with fields for wallet address/payment pointer and amount and a renew monthly toggle (default is off).' style='max-width:300px' class='img-outline'/>
+  5. Enter the amount to make available for sending Web Monetization payments. If you choose not to renew monthly, you'll need to manually [add more funds](#adjust-your-budget) when your balance is zero.
```

Comment:
To be clear: we'll show user is out of funds automatically as well (extension will have orange exclamation icon, and opening it will show you need to add funds). The linked `#adjust-your-budget` section is for manual updates, and can use it even before you're out of funds.

This is the "out of funds" _error_ screen:
<img alt="" src="https://github.com/user-attachments/assets/dc06d43c-b12f-4fd2-ba1b-e8982b643005" width="280">

The message is different depending on whether the earlier budget was one-time/recurring and if now it's one-time/recurring. e.g. 
<img alt="" src="https://github.com/user-attachments/assets/f18890a0-6402-4516-be3f-da4e882dc3ac" width="280">

Reply:
Instead of having these details in the setup step, I've changed the wording of the step a bit and added a new piece to the "Adjust your budget" section. "Set up continuous payments " has an "about" section, so I'm adding one for the budget.